.devcontainer/devcontainer.json

```diff
@@ -10,6 +10,7 @@
     "visualstudioexptteam.vscodeintellicode",
     "esbenp.prettier-vscode"
   ],
+  "mounts": [ "type=volume,target=/var/lib/docker" ],
```

Comment:
Is this working when you use "Clone in volume"?

Reply:
What is "Clone in volume"? What is the use case exactly?

Reply:
That's one of the 2 ways of opening a devcontainer.
@pvizeli uses that way I think

Reply:
I guess its this option?

![image](https://user-images.githubusercontent.com/34061/144419363-7075aafc-cda7-4c34-9275-948207e81243.png)

Cloned my branch using that feature, and it seems to work. I think that feature conflicts with the `workspaceMount` option, but it seems fine with `mounts`.

See: https://code.visualstudio.com/docs/remote/devcontainerjson-reference#_devcontainerjson-properties

Reply:
I have to check if that work on windows, because I use all time this clone function

Reply:
@pvizeli so this seems to work for @mdegat01 on Apple M1. Have you had a chance to check it on your end?

Reply:
Yea this tweak was huge. "Run supervisor" task was timing out (30 seconds) and had to be re-run then taking a good bit to get through the rest of the script and startup before this change. After I think the entire task took maybe 30 seconds.

Reply:
Still on my list